src/java.base/share/classes/sun/security/jca/Providers.java

```diff
@@ -115,7 +129,13 @@ public static Provider getSunProvider() {
      */
     public static Object startJarVerification() {
         ProviderList currentList = getProviderList();
-        ProviderList jarList = currentList.getJarList(jarVerificationProviders);
+        ProviderList jarList;
+        if (RestrictedSecurity.isEnabled()) {
+            jarList = currentList.getJarList(jarVerificationProvidersRS);
+        } else {
+            jarList = currentList.getJarList(jarVerificationProviders);
+        }
+
```

Comment:
Suggest either capturing the argument to `getJarList()` in a local, or simply making it conditional
```java
        ProviderList jarList = currentList.getJarList(
                RestrictedSecurity.isEnabled()
                        ? restrictedJarVerificationProviders
                        : jarVerificationProviders);
```
We don't need blank line 138.

Reply:
Changed to suggestion.